app/src/main/java/com/rentminder/MainActivity.kt

```diff
@@ -188,7 +191,8 @@ fun EditBillAmounts() {
                         .height(55.dp),
                     textStyle = TextStyle.Default.copy(fontSize = 18.sp))
                 SaveRemindButton(
-                    enabled = electricBill.isNotEmpty(), inputEdited = electricBillEdited
+                    enabled = electricBill.isNotEmpty(),
+                    inputEdited = electricBillEdited
```

Comment:
Formatting can, and should, be performed automatically, and consistently, by the IDE.  Formatting code is really programmer's preference anyway, and doesn't necessarily decrease technical debt.